app/Models/Entry.php

```diff
@@ -187,6 +187,7 @@ public function content(bool $withEnclosures = true, bool $allowDuplicateEnclosu
 			$thumbnails = $enclosure['thumbnails'] ?? [];
 			$etitle = $enclosure['title'] ?? '';
 
+			$content .= "\n";
```

Comment:
@Alkarex  How is it important for the Fever API?

Reply:
Not much, but we are concatenating a `<figure ...>...` HTML code right after, and that made the output code difficult to read. Furthermore, I cannot remember for sure, but I believe that having no white-space at all can sometimes lead to a different rendering for some styling rules.